image-processing/v1.py

```diff
@@ -146,16 +144,58 @@ def find_lines_fourier(self, view_plot=True):
         
         return max_positions
 
+    def cwt(self, view_plot=True):
+        x = linefinder.blur_sample_gauss(self,False)
+        widths = (np.arange(1,np.size(x[200])+1, 1))
+        peak_positions = find_peaks_cwt(x[200], widths)
 
+        if view_plot == True:
+            fig, ax = plt.subplots(ncols=3, nrows=1)
+            fig.suptitle('Detecting lines though CWT transform')
+            ax[0].imshow(x, cmap='gray')
+            ax[0].set(xlabel='', ylabel='', title = 'Blurred Sample, sigma = {}'.format(self.sigma))
 
+            ax[1].plot(np.arange(0,np.size(x[200]), 1), x[200])
+            ax[1].set(xlabel='', ylabel='', title = 'Values along 200th row \n for the blurred sample')
 
+            ax[2].imshow(self.original, cmap='gray')
+            ax[2].vlines(peak_positions, color = 'red', ymin=0, ymax=500, linewidth=1)
+            ax[2].set(xlabel='', ylabel='', title='Detected lines')
 
+            plt.show()
+        
+    def scipy_peaks(self, view_plot=True):
+        x = linefinder.blur_sample_gauss(self, False)
+        peak_positions = find_peaks(x[200])
 
+        if view_plot == True:
+            fig, ax = plt.subplots(ncols=3, nrows=1)
+            fig.suptitle('Detecting lines though Scipy find_peaks')
+            ax[0].imshow(x, cmap='gray')
+            ax[0].set(xlabel='', ylabel='', title = 'Blurred Sample, sigma = {}'.format(self.sigma))
+            ax[1].plot(np.arange(0,np.size(x[200]), 1), x[200])
+            ax[1].set(xlabel='', ylabel='', title = 'Values along 200th row \n for the blurred sample')
 
+            ax[2].imshow(self.original, cmap='gray')
+            ax[2].vlines(x=peak_positions[0], color = 'red', ymin=0, ymax=len(x), linewidth=1)
+            ax[2].set(xlabel='', ylabel='', title='Detected lines')
 
+            plt.show()
 
+        return peak_positions[0]
 
 
+'''

```

Comment:
these lines are currently ''' d out as currently haven't defined the baseline or out of 10 classification yes